SocialCareCaseViewerApi/V1/Infrastructure/SccvDbContext.cs

```diff
@@ -18,29 +18,29 @@ public class SccvDbContext : ISccvDbContext
         public IMongoCollection<BsonDocument> matProcessCollection { get; set; }
         public SccvDbContext()
         {
-            string cAfile = Environment.GetEnvironmentVariable("RDS_CA_2019");
```

Comment:
please revert this file back to what's currently in master. This was a temporary "fix" to get it running locally on Mac